packages/core/src/components/forms/_common.scss

```diff
@@ -100,6 +102,10 @@ $control-group-stack: (
   transition: $input-transition;
   vertical-align: middle;
 
+  .bp5-focus-enhanced-contrast & {
+    outline: none;
```

Comment:
it might be better to just save this for a breaking change in a future release, I'm a bit worried about the class based approach since there will be specificity differences between the class based approach, and later changing the color without the class